step_definitions/editTask-steps.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+const navigationLeftPage = require('../page-objects/pages/NavigationLeftPage.js');
+const homeworkListPage = require('../page-objects/pages/HMWRKHomeworkListPage.js');
+const homeworkPage = require('../page-objects/pages/HMWRKHomeworkPage.js');
+const editHomeworkPage = require('../page-objects/pages/HMWRKAddEditHomeworkPage.js');
+
+const { Then } = require('cucumber');
+const { expect } = require('chai');
```

Comment:
const { Then } = require('cucumber');
const { expect } = require('chai');
remove it, it has globally access